vendor/modules.txt

```diff
@@ -9,7 +9,7 @@ cloud.google.com/go/internal/version
 # cloud.google.com/go/storage v1.16.0
 ## explicit
 cloud.google.com/go/storage
-# github.com/Azure/azure-sdk-for-go v55.6.0+incompatible
+# github.com/Azure/azure-sdk-for-go v56.0.0+incompatible
```

Comment:
this major SDK upgrade needs to be done in a separate PR and then this upgraded - can we pull this out?

Reply:
Hi Tom, thanks, I'll only update azure-sdk-for-go in this PR.